src/test-utils/src/drivers/render-test-driver.js

```diff
@@ -35,21 +35,14 @@ const SERVER_CONFIG = {
 };
 
 export default class RenderTestDriver extends BrowserDriver {
-  // TODO - move this method to probe.gl/BrowserDriver
-  waitForFunction(name) {
-    return new Promise(resolve => {
-      this.page.exposeFunction(name, resolve);
-    });
-  }
-
   run(config = SERVER_CONFIG) {
     this.console.log(addColor('Running rendering tests in Chrome instance...', COLOR.YELLOW));
     this.time = Date.now();
     return Promise.resolve()
       .then(_ => this.startServer(config))
       .then(_ => this.startBrowser())
       .then(_ => this.newPage())
-      .then(_ => this.waitForFunction('renderTestComplete'))
+      .then(_ => this.waitForBrowserMessage('renderTestComplete'))
```

Comment:
where do we send out 'renderTestComplete' message?

Reply:
Just above your other comment. `renderTestComplete` becomes available as `window.renderTestComplete`